client/components/fields/editor/ExportTemplate.tsx

```diff
@@ -23,6 +24,7 @@ export class EditorFieldExportTemplateComponent extends React.PureComponent<IPro
             this.props.templates :
             this.props.templates.filter(
                 (filter) => filter.type === this.props.itemType
+                    || ITEM_TYPE.EVENT || ITEM_TYPE.PLANNING
```

Comment:
Won't this also provide `Planning` templates in the list when the filter is for `Events` (and vice-versa)?
Considering the problem is with Events/Event, maybe something like this would suffice:

```js
(filter) => (
    filter.type === this.props.itemType ||
    (filter.type === ITEM_TYPE.EVENT && this.props.itemType === 'event')
)
```

Reply:
Good Catch.